lib/components/private/buildDataAttributes.ts

```diff
@@ -1,6 +1,9 @@
-import mapKeys from 'lodash/mapKeys';
-
 export type DataAttributeMap = Record<string, string | number>;
 
-export default (data: DataAttributeMap = {}) =>
-  mapKeys(data, (_, attrName) => `data-${attrName}`);
+export default (
+  data: DataAttributeMap = {},
+): { [dataAttribute: string]: string } =>
+  Object.assign(
```

Comment:
Any risk that consumers might not have polyfilled this? Should we add this in the release notes?

Reply:
Good question. How about I switch to `object-assign` which react uses internally?

Reply:
Either that, or just spread into a new literal maybe? Given we are spreading below.

Reply:
Changed to a `for of` implementation